liblouis/internal.h

```diff
@@ -210,6 +211,7 @@ typedef struct {
 	TranslationTableOffset definitionRule;
 	TranslationTableOffset otherRules;
 	TranslationTableCharacterAttributes attributes;
+	TranslationTableOffset offset;
```

Comment:
Note that for some very obscure reason I could not put this field (or any field) before `attributes`. Several tests would fail.

Reply:
That is weird. Some kind of alignment problem? Have you tried with -O0?

Reply:
That is what I also thought. But I can't figure out why. No, haven't tried with `-O0`, what does that do again?

Reply:
`-O0` disables all optimization. When I grep for it I see that all the Makefiles set it to `CFLAGS = -g -O2`. TBH I don't know where that is coming from. Have you tried with `./configure CFLAGS='-g -O0'` and then `make`?